packages/universal-router-sdk/src/SwapRouterPancake.ts

```diff
@@ -0,0 +1,67 @@
+import { TradeType } from '@pancakeswap/sdk'
+import { SmartRouter, SmartRouterTrade } from '@pancakeswap/smart-router/evm'
+import { MethodParameters } from '@pancakeswap/v3-sdk'
+import invariant from 'tiny-invariant'
+import { Hex, encodeFunctionData, toHex } from 'viem'
+import abi from './abis/UniversalRouter.json'
```

Comment:
please use typescript and add `as const`

Reply:
do you mean for the abi